dash/frontend/src/app/modules/private/private.module.ts

```diff
@@ -269,6 +270,7 @@ import {FalcoRelatedEventsTableComponent} from './pages/falco/falco-related-even
       MatExpansionModule,
       MatButtonToggleModule,
       NgOptimizedImage,
+      MatSnackBarModule,
```

Comment:
If it's in the app module I don't think it also needs to be in this module

Reply:
It complained if it's not in the private/ public module.